openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -216,16 +217,35 @@ def _filter_fields(self, field_whitelist, serialized_account):
         return visible_serialized_account
 
 
+class UserAccountDisableHistorySerializer(serializers.ModelSerializer):
+    """
+    Class that serializes User account disable history
+    """
+
+    class Meta(object):
+        model = UserAccountDisableHistory
+        fields = ("created", "comment", "disabled")
+
+
 class AccountUserSerializer(serializers.HyperlinkedModelSerializer, ReadOnlyFieldsSerializerMixin):
     """
     Class that serializes the portion of User model needed for account information.
     """
+    disable_history = serializers.SerializerMethodField()
+
     class Meta(object):
         model = User
-        fields = ("username", "email", "date_joined", "is_active")
-        read_only_fields = ("username", "email", "date_joined", "is_active")
+        fields = ("username", "email", "date_joined", "is_active", "disable_history")
+        read_only_fields = ("username", "email", "date_joined", "is_active", "disable_history")
         explicit_read_only_fields = ()
 
+    def get_disable_history(self, user):
+        """ Return a list of the user's disable history """
```

Comment:
docstring format is not edX coding compliant.

Reply:
Yes. I followed the format being used throughout the file to make it consistent